tests/dialstdio.go

```diff
@@ -113,13 +113,10 @@ func testDialStdio(t *testing.T, sb integration.Sandbox) {
 		require.Equal(t, "world", string(dt))
 	}
 
-	t.Run("conn=netpipe", func(t *testing.T) {
-		t.Parallel()
-		do(t, func(t *testing.T, cmd *exec.Cmd) net.Conn {
-			c1, c2 := net.Pipe()
-			cmd.Stdin = c1
-			cmd.Stdout = c1
-			return c2
-		})
+	do(t, func(t *testing.T, cmd *exec.Cmd) net.Conn {
```

Comment:
@cpuguy83 I don't exactly get what the original intention was as it only gets called once and nothing runs in the main test. The subtest itself is not an issue, but the parallel part is.

Reply:
🤷 Probably something like "we can add more conn types" or something.
But yeah it's probably not necessary.

What's wrong with parallel here?

Reply:
> What's wrong with parallel here?

Sandbox (eg. buildkitd daemon started for it and `sb.Context()`) is shut down when the main test ends (in a defer). Parallel moves the subtest to a goroutine where it tries to access the daemon again after the `defer` is already called.